.github/ISSUE_TEMPLATE/submission.md

```diff
@@ -0,0 +1,68 @@
+---
+name: Review submission
+about: Submit an existing lesson for review and acceptance to The Carpentries Lab
+title: '[REV]: '
+labels: new-submission
+assignees: ["tobyhodges", "MCMaurer"]
```

Comment:
Maybe remove this, because we will get notifications anyway and only want to assign whoever is up for editorial role this time?